### PR TITLE
Add New Featured Artist: AAAA news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
🅰️ 🅰️ 🅰️ 🅰️

### CODE REVIEW

File: news/2022-09-11-new-featured-artist-aaaa.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+layout: post
+title: "New Featured Artist: AAAA"
+date: 2022-09-11 16:30:00 +0000
+---
+
+WELCOME **AAAA** TO OUR FEATURED ARTIST LIBRARY.
+
+![](https://assets.ppy.sh/artists/301/header.jpg)
+
+Earning the spotlight in games like *Arcaea*, *maimai*, *CHUNITHM*, *pop'n music*, SDVX*, *Muse Dash*, and... more than we can reasonably list, **AAAA** is an undeniable legend in the realm of rhythm game compositions.
```
Comment (Walavouchey) [inline news/2022-09-11-new-featured-artist-aaaa.md:11]:
```suggestion
Earning the spotlight in games like *Arcaea*, *maimai*, *CHUNITHM*, *pop'n music*, *SDVX*, *Muse Dash*, and... more than we can reasonably list, **AAAA** is an undeniable legend in the realm of rhythm game compositions.
```

**Review by peppy — APPROVED** [review]
not awake enough to give a full review but here's an approve

File: news/2022-09-11-new-featured-artist-aaaa.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+**22** insanely awesome tracks are ready for mapping from [**AAAA**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/301)!
+
+**AAAA**'s Featured Artist reveal was a but unorthodox. In collaboration with fellow legends [Hino Isuka](https://osu.ppy.sh/beatmaps/artists/272) and [Umebsohi Chazuke](https://osu.ppy.sh/beatmaps/artists/98), **AAAA** helped create the custom song [*steganography*](LINK) for the MWC 4K 2022 Tiebreaker!
```
Comment (Walavouchey) [inline news/2022-09-11-new-featured-artist-aaaa.md:15]:
```suggestion
**AAAA**'s Featured Artist reveal was a bit unorthodox. In collaboration with fellow legends [Hino Isuka](https://osu.ppy.sh/beatmaps/artists/272) and [Umeboshi Chazuke](https://osu.ppy.sh/beatmaps/artists/98), **AAAA** helped create the custom song [*steganography*](LINK) for the MWC 4K 2022 tiebreaker!
```

**Comment by Ephemeralis** [comment]
May need to skip CI on this one due to periods in the song titles.

File: news/2022-09-11-new-featured-artist-aaaa.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/301/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### aa...hisuichazuke... - steganography
+
+Play through [the MWC 4K 2022 Tiebreaker](https://osu.ppy.sh/beatmapsets/1846639#mania/3793375) for yourself, mapped by [-mint-](https://osu.ppy.sh/users/8976576), [guden](https://osu.ppy.sh/users/11626065), [MyZterioN-](https://osu.ppy.sh/users/8521723), and [Toaph Daddy](https://osu.ppy.sh/users/7616811)!
```
Comment (Walavouchey) [inline news/2022-09-11-new-featured-artist-aaaa.md:27]:
```suggestion
Play through [the MWC 4K 2022 tiebreaker](https://osu.ppy.sh/beatmapsets/1846639#mania/3793375) for yourself, mapped by [-mint-](https://osu.ppy.sh/users/8976576), [guden](https://osu.ppy.sh/users/11626065), [MyZterioN-](https://osu.ppy.sh/users/8521723), and [Toaph Daddy](https://osu.ppy.sh/users/7616811)!
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2022-09-11-new-featured-artist-aaaa.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+WELCOME **AAAA** TO OUR FEATURED ARTIST LIBRARY.
+
+![](https://assets.ppy.sh/artists/301/header.jpg)
+
+Earning the spotlight in games like *Arcaea*, *maimai*, *CHUNITHM*, *pop'n music*, *SDVX*, *Muse Dash*, and... more than we can reasonably list, **AAAA** is an undeniable legend in the realm of rhythm game compositions.
+
+**22** insanely awesome tracks are ready for mapping from [**AAAA**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/301)!
+
+**AAAA**'s Featured Artist reveal was a bit unorthodox. In collaboration with fellow legends [Hino Isuka](https://osu.ppy.sh/beatmaps/artists/272) and [Umeboshi Chazuke](https://osu.ppy.sh/beatmaps/artists/98), **AAAA** helped create the custom song [*steganography*](LINK) for the MWC 4K 2022 tiebreaker!
```
Comment (LeoFLT) [inline news/2022-09-11-new-featured-artist-aaaa.md:15]: It's missing a link here (to the song file/listing?)

Comment (mangomizer) [reply news/2022-09-11-new-featured-artist-aaaa.md:15]: unsure if markdown supports greek letters but the e in steganography is actually epsilon "ε"

Comment (Walavouchey) [reply news/2022-09-11-new-featured-artist-aaaa.md]: iirc news posts use the romanised title

**Review by peppy — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**